From 1527157973d76838d3fe1cb04d5a70fd78f4145d Mon Sep 17 00:00:00 2001
From: undoLogic <info@setupcase.com>
Date: Mon, 25 Aug 2025 03:59:51 -0400
Subject: [PATCH] 1. setupcase - adjustment

---
 sourceFiles/src/Controller/AppController.php | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sourceFiles/src/Controller/AppController.php b/sourceFiles/src/Controller/AppController.php
index 809c93d..9dc6cff 100644
--- a/sourceFiles/src/Controller/AppController.php
+++ b/sourceFiles/src/Controller/AppController.php
@@ -17,6 +17,7 @@ declare(strict_types=1);
 namespace App\Controller;
 
 use Cake\Controller\Controller;
+use Cake\Event\EventInterface;
 
 /**
  * Application Controller
@@ -50,4 +51,9 @@ class AppController extends Controller
          */
         //$this->loadComponent('FormProtection');
     }
+
+
+    public function beforeFilter(EventInterface $event) {
+        dd('appController added');
+    }
 }
-- 
2.47.1.windows.1

